backend/functions/ticket.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+
+function getTicket($data,$con){
+    $ticket = selectStatement($con, 'ticket', true, 'id=?', [$data['ticketId']]);
+    foreach ($ticket as $key => $value) {
+        $returnArray = [
+            'id'            => $value['id'],
+            'title'         => $value['title'],
+            'description'   => $value['description'],
+            'status'        => $value['status'],
+            'worktime'      => $value['worktime'],
+            'is_archived'   => $value['is_archived'],
+            'date_created'  => $value['date_created'] 
+        ];
+    }
+    return $returnArray;
+}
+
+function getTickets($data,$con){
+    $returnArray = [];
+
+    $tickets = selectStatement($con,'ticket',false,null,null);
+    foreach ($tickets as $key => $value) {
```

Comment:
meerdere foreach = andere naamgeving $key en $value voor betere overzicht van wat het is